[2024-11-02T23:01:01.454+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-02T23:01:01.464+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_sales_data_to_db.fetch_sales_data_raw scheduled__2024-11-02T23:00:00+00:00 [queued]>
[2024-11-02T23:01:01.470+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_sales_data_to_db.fetch_sales_data_raw scheduled__2024-11-02T23:00:00+00:00 [queued]>
[2024-11-02T23:01:01.470+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-02T23:01:01.479+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): fetch_sales_data_raw> on 2024-11-02 23:00:00+00:00
[2024-11-02T23:01:01.483+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=613) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-02T23:01:01.484+0000] {standard_task_runner.py:72} INFO - Started process 618 to run task
[2024-11-02T23:01:01.485+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'load_sales_data_to_db', 'fetch_sales_data_raw', 'scheduled__2024-11-02T23:00:00+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/raw/sales.py', '--cfg-path', '/tmp/tmp8c9vy2sm']
[2024-11-02T23:01:01.486+0000] {standard_task_runner.py:105} INFO - Job 155: Subtask fetch_sales_data_raw
[2024-11-02T23:01:01.522+0000] {task_command.py:467} INFO - Running <TaskInstance: load_sales_data_to_db.fetch_sales_data_raw scheduled__2024-11-02T23:00:00+00:00 [running]> on host f05559d9e656
[2024-11-02T23:01:01.573+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='load_sales_data_to_db' AIRFLOW_CTX_TASK_ID='fetch_sales_data_raw' AIRFLOW_CTX_EXECUTION_DATE='2024-11-02T23:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-02T23:00:00+00:00'
[2024-11-02T23:01:01.574+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-02T23:01:01.574+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-02T23:01:01.574+0000] {logging_mixin.py:190} INFO - Current task name:fetch_sales_data_raw state:running start_date:2024-11-02 23:01:01.464866+00:00
[2024-11-02T23:01:01.575+0000] {logging_mixin.py:190} INFO - Dag name:load_sales_data_to_db and current dag run status:running
[2024-11-02T23:01:01.575+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-02T23:01:02.710+0000] {sales.py:49} INFO - Successfully fetched sales data from the API.
[2024-11-02T23:01:02.720+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-02T23:01:02.727+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-02T23:01:02.728+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=load_sales_data_to_db, task_id=fetch_sales_data_raw, run_id=scheduled__2024-11-02T23:00:00+00:00, execution_date=20241102T230000, start_date=20241102T230101, end_date=20241102T230102
[2024-11-02T23:01:02.734+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-02T23:01:02.735+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-02T23:01:02.735+0000] {logging_mixin.py:190} INFO - Dag name:load_sales_data_to_db queued_at:2024-11-02 23:01:00.598843+00:00
[2024-11-02T23:01:02.735+0000] {logging_mixin.py:190} INFO - Task hostname:f05559d9e656 operator:PythonOperator
[2024-11-02T23:01:02.752+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-02T23:01:02.765+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-02T23:01:02.769+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
